package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "devDependencies": {
+    "@4tw/cypress-drag-drop": "^2.2.1",
+    "cypress": "^10.4.0",
+    "cypress-file-upload": "^5.0.8",
+    "cypress-iframe": "^1.0.1",
+    "prettier": "^2.7.1"
```

Comment:
Do you think we can/should remove prettier from here, as we already have it in pre-commit? Right now we have the same version in both, but in case we change it